stable/minecraft/Chart.yaml

```diff
@@ -1,5 +1,7 @@
 name: minecraft
-version: 0.2.0
+version: 0.2.1
+appVersion: latest
```

Comment:
I don't think this is a valid value for appVersion. I'd recommend simply checking what the current latest is and using that value.

Reply:
In the values.yaml: 
 # One of: LATEST, SNAPSHOT, or a specific version (ie: "1.7.9").
  version: "LATEST"
So I think we can use 1.7.9